Can't open external links to projects in subgroups

### I confirm that I have searched the issue tracker
Before opening an issue, take a few minutes to search the issue tracker (both open and closed issues) for the problem you are facing. This will reduce the need for issue triage and save time.

 - [x] I confirm that I have searched the issue tracker

### Describe the bug

LabNex URLs to projects in a subgroup (e.g. `/<group>/<subgroup>/<project>` or any further nesting) aren't recognised.

### To Reproduce

1. Open `labnex://gitlab.com/gitlab-org/gitlab` (https://gitlab.com/gitlab-org/gitlab)
    - Successfully opens the project
2. Open `labnex://gitlab.com/gitlab-org/auth/glgo` (https://gitlab.com/gitlab-org/auth/glgo)
    - Opens the dashboard home screen

### Expected behavior

Both projects can be opened via URL.

### Versions (please complete the following information)**
 - LabNex: 7.0.0
 - Android: 13
 - GitLab: Any

### GitLab instance
 - [x] gitlab.com
 - [ ] framagit.org
 - [x] Self-hosted instance
 - [ ] Other service (Name/URL: )

### Mode of installation
 - [ ] Google Play Store
 - [x] F-droid
 - [ ] APK
 - [ ] Built from source
 - [ ] OpenAPK
 - [ ] Other stores (Name: )

### Declaration
- [x] I have carefully read and agree to follow the [contribution guidelines](https://github.com/labnex/LabNex/wiki/Contributing).

I just started looking at implementing this, but I think ~~the unreleased changes around groups~~ are breaking the link handler:

<details>
<summary>Crash log</summary>

```
FATAL EXCEPTION: main
Process: com.labnex.app, PID: 18739
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.labnex.app/com.labnex.app.activities.DeepLinksActivity}: android.view.InflateException: Binary XML file line #8 in com.labnex.app:layout/activity_deeplinks: Binary XML file line #8 in com.labnex.app:layout/activity_deeplinks: Error inflating class androidx.coordinatorlayout.widget.CoordinatorLayout
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3864)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4006)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:111)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2462)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:240)
	at android.os.Looper.loop(Looper.java:351)
	at android.app.ActivityThread.main(ActivityThread.java:8377)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
Caused by: android.view.InflateException: Binary XML file line #8 in com.labnex.app:layout/activity_deeplinks: Binary XML file line #8 in com.labnex.app:layout/activity_deeplinks: Error inflating class androidx.coordinatorlayout.widget.CoordinatorLayout
Caused by: android.view.InflateException: Binary XML file line #8 in com.labnex.app:layout/activity_deeplinks: Error inflating class androidx.coordinatorlayout.widget.CoordinatorLayout
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
	at android.view.LayoutInflater.createView(LayoutInflater.java:857)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1009)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:964)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:663)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
	at com.labnex.app.databinding.ActivityDeeplinksBinding.inflate(ActivityDeeplinksBinding.java:98)
	at com.labnex.app.databinding.ActivityDeeplinksBinding.inflate(ActivityDeeplinksBinding.java:92)
	at com.labnex.app.activities.DeepLinksActivity.onCreate(DeepLinksActivity.java:45)
	at android.app.Activity.performCreate(Activity.java:8397)
	at android.app.Activity.performCreate(Activity.java:8370)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1403)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3837)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4006)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:111)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2462)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:240)
	at android.os.Looper.loop(Looper.java:351)
	at android.app.ActivityThread.main(ActivityThread.java:8377)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 13: TypedValue{t=0x2/d=0x7f040428 a=-1}, theme={InheritanceMap=[id=0x7f13000bcom.labnex.app:style/AppTheme, id=0x7f1302a1com.labnex.app:style/Theme.Material3.Light.NoActionBar, id=0x7f13029bcom.labnex.app:style/Theme.Material3.Light, id=0x7f130062com.labnex.app:style/Base.Theme.Material3.Light, id=0x7f1300b7com.labnex.app:style/Base.V24.Theme.Material3.Light, id=0x7f130093com.labnex.app:style/Base.V14.Theme.Material3.Light, id=0x7f1302c0com.labnex.app:style/Theme.MaterialComponents.Light, id=0x7f130071com.labnex.app:style/Base.Theme.MaterialComponents.Light, id=0x7f1300accom.labnex.app:style/Base.V21.Theme.MaterialComponents.Light, id=0x7f13009bcom.labnex.app:style/Base.V14.Theme.MaterialComponents.Light, id=0x7f13009ccom.labnex.app:style/Base.V14.Theme.MaterialComponents.Light.Bridge, id=0x7f13015acom.labnex.app:style/Platform.MaterialComponents.Light, id=0x7f130277com.labnex.app:style/Theme.AppCompat.Light, id=0x7f130055com.labnex.app:style/Base.Theme.AppCompat.Light, id=0x7f1300bdcom.labnex.app:style/Base.V28.Theme.AppCompat.Light, id=0x7f1300bacom.labnex.app:style/Base.V26.Theme.AppCompat.Light, id=0x7f1300b4com.labnex.app:style/Base.V23.Theme.AppCompat.Light, id=0x7f1300b2com.labnex.app:style/Base.V22.Theme.AppCompat.Light, id=0x7f1300a8com.labnex.app:style/Base.V21.Theme.AppCompat.Light, id=0x7f1300c0com.labnex.app:style/Base.V7.Theme.AppCompat.Light, id=0x7f130157com.labnex.app:style/Platform.AppCompat.Light, id=0x7f130162com.labnex.app:style/Platform.V25.AppCompat.Light, id=0x1030241android:style/Theme.Material.Light.NoActionBar, id=0x1030237android:style/Theme.Material.Light, id=0x103000candroid:style/Theme.Light, id=0x1030005android:style/Theme], Themes=[com.labnex.app:style/AppTheme, forced, com.labnex.app:style/Theme.AppCompat.Empty, forced, android:style/Theme.DeviceDefault.Light.DarkActionBar, forced]}
	at android.content.res.TypedArray.getDrawableForDensity(TypedArray.java:1007)
	at android.content.res.TypedArray.getDrawable(TypedArray.java:991)
	at android.view.View.<init>(View.java:5616)
	at android.view.ViewGroup.<init>(ViewGroup.java:707)
	at android.view.ViewGroup.<init>(ViewGroup.java:703)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.<init>(CoordinatorLayout.java:216)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.<init>(CoordinatorLayout.java:211)
	... 26 more
```
</details>

Relatedly, it would be great if you could support debug builds with a separate application ID, so I don't need to uninstall the release version to test changes.  (I think you can just set `applicationIdSuffix` for the debug build?)

Edit: Looks like a clean debug build of 7.0.0 also fails with this error when launching from a `labnex://` URL for me 🤔

--- a/app/src/main/java/com/labnex/app/activities/DeepLinksActivity.java
+++ b/app/src/main/java/com/labnex/app/activities/DeepLinksActivity.java
@@ -132,15 +132,13 @@ public class DeepLinksActivity extends BaseActivity {
 	private void processDeepLink() {
 		List<String> pathSegments = data.getPathSegments();
 
-		if (pathSegments.size() >= 5) {
-			String sectionType = pathSegments.get(3);
-
-			String userOrGroupName = pathSegments.get(0);
-			String projectName = pathSegments.get(1);
-			String projectPathWithNamespace = userOrGroupName + "/" + projectName;
+		int split = pathSegments.indexOf("-");
+		if (split >= 2) {
+			String sectionType = pathSegments.get(split + 1);
+			String projectPathWithNamespace = String.join("/", pathSegments.subList(0, split));
 
 			if ("issues".equals(sectionType)) {
-				String issueIdStr = pathSegments.get(4);
+				String issueIdStr = pathSegments.get(split + 2);
 				if (issueIdStr != null && issueIdStr.matches("\\d+")) {
 					int issueIid = Integer.parseInt(issueIdStr);
 					goToSpecificIssue(projectPathWithNamespace, issueIid);
@@ -148,7 +146,7 @@ public class DeepLinksActivity extends BaseActivity {
 					goToProjectWithSection(projectPathWithNamespace, "issue");
 				}
 			} else if ("merge_requests".equals(sectionType)) {
-				String mrIdStr = pathSegments.get(4);
+				String mrIdStr = pathSegments.get(split + 2);
 				if (mrIdStr != null && mrIdStr.matches("\\d+")) {
 					int mergeRequestIid = Integer.parseInt(mrIdStr);
 					goToSpecificMergeRequest(projectPathWithNamespace, mergeRequestIid);
@@ -159,9 +157,7 @@ public class DeepLinksActivity extends BaseActivity {
 				goToProject(projectPathWithNamespace);
 			}
 		} else if (pathSegments.size() >= 2) {
-			String userOrGroupName = pathSegments.get(0);
-			String projectName = pathSegments.get(1);
-			String projectPathWithNamespace = userOrGroupName + "/" + projectName;
+			String projectPathWithNamespace = String.join("/", pathSegments);
 			goToProject(projectPathWithNamespace);
 		} else {
 			startActivity(mainIntent);

--- a/app/src/main/java/com/labnex/app/activities/DeepLinksActivity.java
+++ b/app/src/main/java/com/labnex/app/activities/DeepLinksActivity.java
@@ -137,7 +137,7 @@ public class DeepLinksActivity extends BaseActivity {
 			String sectionType = pathSegments.get(split + 1);
 			String projectPathWithNamespace = String.join("/", pathSegments.subList(0, split));
 
-			if ("issues".equals(sectionType)) {
+			if ("issues".equals(sectionType) || "work_items".equals(sectionType)) {
 				String issueIdStr = pathSegments.get(split + 2);
 				if (issueIdStr != null && issueIdStr.matches("\\d+")) {
 					int issueIid = Integer.parseInt(issueIdStr);

